Scripts/DCS-BIOS/lib/Protocol.lua

```diff
@@ -187,6 +188,11 @@ local lastFrameTime = LoGetModelTime()
 local updateCounter = 0
 local updateSkipCounter = 0
 function BIOS.protocol.step()
+
+	if( metadataStartModule == nil or  metadataEndModule == nil) then
+		return -- this should never happen since init() is being called but it removes intellisense warnings

```

Comment:
I'd suggest throwing an error if this happens, because that seems pretty bad.

Reply:
```2023-09-29 16:55:02.215 ERROR   Lua::Config (Main): Call error LuaExportAfterNextFrame:[string "C:\Users\.......\Saved Games\DCS\Scripts/DCS-BIOS/lib/Protocol.lua"]:194: Either MetadataStart or MetadataEnd was nil.
stack traceback:```